src/jit/utils.cpp

```diff
@@ -1194,6 +1194,9 @@ void HelperCallProperties::init()
 
         case CORINFO_HELP_NEW_CROSSCONTEXT:
         case CORINFO_HELP_NEWFAST:
+#ifdef FEATURE_READYTORUN_COMPILER
```

Comment:
I don't think that we shodul be using #ifdef READYTORUN
we  should just remove these ifdefs as these values are unconditionally defined in corinfo.h
I also note that the current source base already contains a mixture of places where we omit these #ifdef's
and instead use  if (opts.IsReadyToRun())
(see Importer.cpp line 4456)